pkg/remoteimage/pull.go

```diff
@@ -68,11 +135,14 @@ func (p puller) Pull(ctx context.Context) (err error) {
 		})
 
 		if err == nil {
+			klog.V(2).Info("remoteimage.Pull(with creds): pulling %s completed with err==nil", p.image.String())
```

Comment:
this is missing a `Infof` instead of Info

Reply:
good catch!